src/backend/distributed/metadata/node_metadata.c

```diff
@@ -150,6 +150,7 @@ PG_FUNCTION_INFO_V1(get_shard_id_for_distribution_column);
 PG_FUNCTION_INFO_V1(citus_nodename_for_nodeid);
 PG_FUNCTION_INFO_V1(citus_nodeport_for_nodeid);
 PG_FUNCTION_INFO_V1(citus_coordinator_nodeid);
+PG_FUNCTION_INFO_V1(citus_is_coordinator);
```

Comment:
Should we maybe introduce `get_current_host_info` or such which returns the the tuple (hostname, hostport, nodeid, is_coordinator)?
(we can keep this as well, and we can do it with a separate pr, mostly asking your thoughts)

Reply:
Well, never mind, we can achieve this with simple SQL queries anyway:
```SQL
SELECT *, 
	CASE WHEN groupid = (SELECT groupid FROM pg_dist_local_group) THEN '*' 
		 ELSE ' ' END current_node 
FROM pg_dist_node;

┌───────────┬──────────┬──────────────┐
│ nodename  │ nodeport │ current_node │
├───────────┼──────────┼──────────────┤
│ localhost │     9700 │              │
│ localhost │     9701 │ *            │
│ localhost │     9702 │              │
│ localhost │     5432 │              │
└───────────┴──────────┴──────────────┘
(4 rows)
```